src/Storage/Service/AbstractUrlStorage.php

```diff
@@ -185,8 +185,8 @@ public function isReadOnly(): bool
         return $this->readOnly;
     }
 
-    public function isToSkip(): bool
+    public function isSkip(): bool
```

Comment:
```suggestion
    public function shouldSkip(): bool
```

"is" for property
"should" for action